episodes/severity-static.Rmd

```diff
@@ -0,0 +1,640 @@
+
+Figures A and B show the cumulative numbers of cases and deaths of SARS, and Figure C shows the observed (biased) CFR estimates as a function of time, i.e. the ratio of the cumulative number of cases to deaths at time $t$. Due to the delay from the onset of symptoms to death, the biased estimate of CFR at time $t$ underestimates the realised CFR at the end of an outbreak (i.e. 302/1755 = 17.2 %). 
+
+![Observed (biased) confirmed case fatality ratio of severe acute respiratory syndrome (SARS) in Hong Kong, 2003.](fig/cfr-pone.0006852.g003-fig_abc.png)
+
+Nevertheless, even by only using the observed data for the period March 19 to April 2, `cfr_static()` can yield an appropriate prediction (Figure D), e.g. the delay-adjusted CFR at March 27 is 18.1 % (95% CI: 10.5, 28.1). An overestimation is seen in the very early stages of the epidemic, but the 95% confidence limits in the later stages include the realised CFR (i.e. 17.2 %).
+
+![Early determination of the delay-adjusted confirmed case fatality ratio of severe acute respiratory syndrome (SARS) in Hong Kong, 2003.](fig/cfr-pone.0006852.g003-fig_d.png)
+
+We can explore this behaviour using the `cfr_rolling()` function. With `tail()`, we show that the latest CFR estimates are equal to the `cfr_static()` outputs:  
+
+```{r}
+# Calculate the rolling daily naive CFR
+rolling_cfr_naive <- cfr::cfr_rolling(data = ebola1976)
+
+tail(rolling_cfr_naive)
+```
+
+```{r}
+# Calculate the rolling daily delay-adjusted CFR
+rolling_cfr_adjusted <- cfr::cfr_rolling(
+  data = ebola1976,
+  delay_density = function(x) density(onset_to_death_ebola, x)
+)
+
+tail(rolling_cfr_adjusted)
+```
+
+Now, let's visualise both results in a time series. How would the naive and delay-adjusted CFR estimates perform in real time?
+
+```{r,echo=FALSE,warning=FALSE,message=FALSE}
+
+# get the latest delay-adjusted CFR
+rolling_cfr_adjusted_end <-
+  rolling_cfr_adjusted %>%
+  dplyr::slice_tail()
+
+# bind by rows both output data frames
+bind_rows(
+  rolling_cfr_naive %>%
+    mutate(method = "naive"),
+  rolling_cfr_adjusted %>%
+    mutate(method = "adjusted")
+) %>%
+  # visualise both adjusted and unadjusted rolling estimates
+  ggplot() +
+  geom_ribbon(
+    aes(
+      date,
+      ymin = severity_low,
+      ymax = severity_high,
+      fill = method
+    ),
+    alpha = 0.2, show.legend = FALSE
+  ) +
+  geom_line(
+    aes(date, severity_mean, colour = method)
+  ) +
+  geom_hline(data = rolling_cfr_adjusted_end,
+             aes(yintercept = severity_mean)) +
+  geom_hline(data = rolling_cfr_adjusted_end,
+             aes(yintercept = severity_low), lty = 2) +
+  geom_hline(data = rolling_cfr_adjusted_end,
+             aes(yintercept = severity_high), lty = 2)
+```
```

Comment:
This figure could be formatted to look more publication ready; e.g. capitalise axis and legend names, place legend correctly, use uniform theme

Reply:
added in #9 for further evaluation